src/transformers/models/aria/image_processing_aria.py

```diff
@@ -244,6 +268,12 @@ def preprocess(
         # All transformations expect numpy arrays.
         images = [to_numpy_array(image) for image in images]
 
+        if is_scaled_image(images[0]) and do_rescale:
```

Comment:
Way faster for `do_rescale=False`

```suggestion
        if do_rescale and is_scaled_image(images[0]):
```